ladybug_comfort/solarcal.py

```diff
@@ -9,14 +9,23 @@
 https://escholarship.org/uc/item/89m1h2dg
 
 [2] ASHRAE Standard 55 (2017). "Thermal Environmental Conditions for Human Occupancy".
+
+Properties:
+    SOLARCALSPLINES: A dictionary with two keys: 'standing' and 'seated'.
+        Each value for these keys is a 2D matrix of projection factors
+        for human geometry.  Each row refers to an degree of azimuth and each
+        colum refers to a degree of altitude.
 """
 from __future__ import division
 
 from ladybug.skymodel import calc_sky_temperature
-from .__init__ import SOLARCALSPLINES
+from ._loadmannequin import load_solarcal_splines
 
 import math
 
+# load the spline data that gets used in solarcal.
+SOLARCALSPLINES = load_solarcal_splines()
```

Comment:
Why don't you just load the values here in this file? Do we need to have it in mannequin module? I don't know if one would look for `solarcal` splines in `loadmannequin` module.

Reply:
Good point.  I guess now that we are not worried about bogging down __init__ with code, I will just move it here.